src/List.js

```diff
@@ -77,77 +129,79 @@ export function List() {
   };
-  if (items.length) {
-    return (
-      <>
-        <label htmlFor="filterItems">Filter items:</label>
-        <input
-          name="filterItems"
-          type="text"
-          value={filterItem}
-          placeholder="Start typing here..."
-          onChange={(event) => setFilterItem(event.target.value)}
-        ></input>
-        <button onClick={() => setFilterItem('')}>X</button>
-        <ul className="list">
-          {items &&
-            items
-              .filter((item) => !!item.id)
-              .filter((item) =>
-                item.name.toLowerCase().includes(filterItem.toLowerCase()),
-              )
-              .map((item) => {
-                return (
-                  <li key={item.id}>
-                    <input
-                      type="checkbox"
-                      id={`custom-checkbox-${item.id}`}
-                      name={item.name}
-                      value={item.name}
-                      checked={
-                        !!item.lastPurchasedDate &&
-                        new Date() - item.lastPurchasedDate < ONE_MINUTE
-                      }
-                      onChange={(e) => handleChange(item.id, e)}
-                    />
-                    <label htmlFor={`custom-checkbox-${item.id}`}>
-                      {item.name}
-                    </label>
-                  </li>
-                );
-              })}
-        </ul>
-        <NavigationMenu />
-      </>
-    );
-  }
-  return (
+
+  return items.length ? (
+    <>
+      <label htmlFor="filterItems">Filter items:</label>
+      <input
+        name="filterItems"
+        type="text"
+        value={filterItem}
+        placeholder="Start typing here..."
+        onChange={(event) => setFilterItem(event.target.value)}
+      ></input>
+      <button onClick={() => setFilterItem('')}>X</button>
+      <ul className="list">
+        {items &&
+          items
+            .filter((item) => !!item.id)
+            .filter((item) =>
+              item.name.toLowerCase().includes(filterItem.toLowerCase()),
+            )
+            .sort(itemSort)
+            .map((item) => {
+              return (
+                <li key={item.id} className={itemState(item)}>
+                  <input
+                    type="checkbox"
+                    aria-label={itemState(item)}
+                    id={`custom-checkbox-${item.id}`}
+                    name={item.name}
+                    value={item.name}
+                    checked={
+                      !!item.lastPurchasedDate &&
+                      new Date() - item.lastPurchasedDate < ONE_MINUTE
+                    }
+                    onChange={(e) => handleChange(item.id, e)}
+                  />
+                  <label htmlFor={`custom-checkbox-${item.id}`}>
+                    {item.name}
+                  </label>
+                </li>
+              );
+            })}
+      </ul>
+      <NavigationMenu />
+    </>
+  ) : (
```

Comment:
Just curious...this was fixed last week according to @MikeBLambert's suggestion. He and @bajancode approved it. What made you decide to change it to a ternary operator?